### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by gawel**

### CODE REVIEW

**Comment by gawel** [comment]
Hm, I messed up with my branches... 8285989 should not be included here. Let me know if it's a problem (should not be...). Sorry for that

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Contribution ares always welcome but please try to follow those few rules:
```
Comment (tonioo) [inline CONTRIBUTING.rst]: are

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- Modoboa try to respect the `PEP8 <https://www.python.org/dev/peps/pep-0008/>`_.
```
Comment (tonioo) [inline CONTRIBUTING.rst]: tries

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- If you're adding some code, try to add some tests too. You can run the tests
+  suite locally by using `tox <https://testrun.org/tox/latest/config.html>`_
+
+There is also a few commands that can help you to contribute:
```
Comment (tonioo) [inline CONTRIBUTING.rst]: There are

**Comment by gawel** [comment]
All good

**Pull request merged**